generated/nidcpower/session.py

```diff
@@ -1546,14 +1546,16 @@ def configure_aperture_time(self, aperture_time, units=enums.ApertureTimeUnits.S
         Args:
             aperture_time (float): Specifies the aperture time. Refer to the *Aperture Time* topic for your
                 device in the *NI DC Power Supplies and SMUs Help* for more information.
+
             units (enums.ApertureTimeUnits): Specifies the units for **apertureTime**.
                 **Defined Values**:
 
-                +----------------------------------------+------------------------------+
-                | NIDCPOWER_VAL_SECONDS (1028)           | Specifies seconds.           |
-                +----------------------------------------+------------------------------+
-                | NIDCPOWER_VAL_POWER_LINE_CYCLES (1029) | Specifies Power Line Cycles. |
-                +----------------------------------------+------------------------------+
+                +--------------------------------------------+------------------------------+
+                | ApertureTimeUnits.SECONDS (1028)           | Specifies seconds.           |
```

Comment:
Is putting the numeric value there pretty standard? It's just noise to the Python user. If easy, we should remove.

Reply:
This is from the documentation. We are not adding that. All we are doing is replacing the C name with the Python name. It would be difficult to find since each driver does things different.